modules/influxdb/src/test/java/org/testcontainers/containers/InfluxDBContainerTest.java

```diff
@@ -1,44 +1,135 @@
 package org.testcontainers.containers;
 
-import org.influxdb.InfluxDB;
-import org.junit.ClassRule;
-import org.junit.Test;
-
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.influxdb.client.InfluxDBClient;
+import com.influxdb.client.QueryApi;
+import com.influxdb.client.WriteApi;
+import com.influxdb.client.domain.Bucket;
+import com.influxdb.client.domain.BucketRetentionRules;
+import com.influxdb.client.domain.WritePrecision;
+import com.influxdb.client.write.Point;
+import com.influxdb.query.FluxRecord;
+import com.influxdb.query.FluxTable;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import org.junit.Test;
+
 public class InfluxDBContainerTest {
 
-    @ClassRule
-    public static InfluxDBContainer<?> influxDBContainer = new InfluxDBContainer<>(
-        InfluxDBTestImages.INFLUXDB_TEST_IMAGE
-    );
+    private static final String USERNAME = "new-test-user";
+
+    private static final String PASSWORD = "new-test-password";
+
+    private static final String ORG = "new-test-org";
+
+    private static final String BUCKET = "new-test-bucket";
+
+    private static final String RETENTION = "1w";
+
+    private static final String ADMIN_TOKEN = "super-secret-token";
+    private static final int SECONDS_IN_WEEK = 604800;
 
     @Test
-    public void getUrl() {
-        String actual = influxDBContainer.getUrl();
+    public void getInfluxDBClient() {
+        try (
+            // constructorWithDefaultVariables {
+            final InfluxDBContainer<?> influxDBContainer = new InfluxDBContainer<>(
+                InfluxDBTestUtils.INFLUXDB_V2_TEST_IMAGE)
```

Comment:
code snippets are using `InfluxDBTestUtils.INFLUXDB_V2_TEST_IMAGE` which is not useful for docs. Just in those cases can we use the string image name, please? Same for `InfluxDBTestUtils.INFLUXDB_V1_TEST_IMAGE`